### DISCUSSION TITLE
Can a Relationship Field be "relationTo" Array Field?

### DISCUSSION THREAD

**Discussion by eygsoft** [payloadcms/payload#9850] [Q&A]
One to many relationships are very practical, so PayloadCMS' Array Field is amazing. However, Relationship Field cannot connect to the table created by Array Field. Is there any solution?

```typescript
export const Questions: CollectionConfig = {
  slug: 'questions',
  fields: [
    { name: 'name', type: 'text' },
    {
      name: 'choices',
      type: 'array',
      fields: [{ name: 'title', type: 'text' }],
    },
  ],
}

export const ChoicesRecords: CollectionConfig = {
  slug: 'choices_records',
  fields: [
    {
      name: 'choice',
      type: 'relationship',
      relationTo: 'questions.choices', // If that's the case, that would be great
    },
    {
      name: 'user',
      type: 'relationship',
      relationTo: 'users',
    },
    {
      name: 'completed_at',
      type: 'date',
    },
  ],
}
```

I know it can be implemented using Join Field and Relationship Field, but it still requires manual sorting and is far less aesthetically pleasing than Array Field.

Thanks a lot!

**Comment by eygsoft** [comment]
The solutions I can think of are:

1. Join Field is very similar to Array Field, you can give Join Field an option to make its Admin the same as Array Field.
2. let Array Field choose to configure Fields or Collection. If Collection is selected, it will be given a Parent ID Field and a Sort Field.

I think this would be a great improvement because one to many relationships are very common. If implemented using Relationships and Join, the functionality is powerful but the Admin operation is cumbersome, while the Admin operation of Array Field is convenient but the functionality is too limited.

Thanks a lot!

**Comment by DanRibbens** [comment]
Check out this example repo https://github.com/payloadcms/ordered-join I think it might give you a workaround for what you're asking for.
